components/zendesk/sources/common/base.mjs

```diff
@@ -0,0 +1,184 @@
+import crypto from "crypto";
+import zendesk from "../../zendesk.app.mjs";
+import constants from "../../constants.mjs";
+
+export default {
+  props: {
+    zendesk,
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+    },
```

Comment:
I might have something set up incorrectly, but I'm getting linter errors for this prop: `eslint(pipedream/props-label)` and `eslint(pipedream/props-description)`

To fix the errors:
```suggestion
    http: "$.interface.http",
```